x/incentives/keeper/distribute.go

```diff
@@ -619,6 +625,10 @@ func (k Keeper) distributeInternal(
 		// 2. To avoid having to do a state migration on existing gauges at time of adding
 		// this change, since prior to this, CL gauges were not required to associate with
 		// an uptime that was authorized.
+		//
+		// Note that this fallback happens even if the gauge is an internal gauge. This is because
+		// we want to be able to unauthorize internal gauges as well in the event that there is an
+		// issue found with a previously authorized uptime.
```

Comment:
A secondary reason is that allowing "supported but unauthorized uptimes" to be valid for internal gauges would require hackily bypassing CL incentive record checks, which currently validate against authorized uptimes. I started with trying this and ended up walking back my impl to this version.